### DISCUSSION TITLE
Guidance for Flux Dev in Forge UI?

### DISCUSSION THREAD

**Discussion by FrankEBailey** [lllyasviel/stable-diffusion-webui-forge#1828] [Q&A] [answered]
On other GUIs and platforms, there's a Guidance setting that's settable between 2 and 5 for Flux Dev. How can I do that in Forge? I'm using flux1-dev-bnb-nf4-v2.safetensors

**Comment by Tsukasuke** [comment, marked as answer]
I think you may be looking for Distilled CFG Scale.  You can find more info here:
https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/981
